assists/v/m.sh

```diff
@@ -4,7 +4,7 @@
 
 amixer -D pulse sset Capture toggle
 
-MICSTATUS=$(amixer get Capture | grep -w -o -e "on\|off" | sort -u | tr "[a-z]" "[A-Z]")
+MICSTATUS=$(amixer -D pulse get Capture | grep -w -o -e "on\|off" | sort -u | tr "[a-z]" "[A-Z]")
```

Comment:
```suggestion
MICSTATUS=$(amixer get Capture | grep -w -o -e "on\|off" | sort -u | tr "[a-z]" "[A-Z]")
if [ -z "$MICSTATUS" ]; then
MICSTATUS=$(amixer -D pulse get Capture | grep -w -o -e "on\|off" | sort -u | tr "[a-z]" "[A-Z]")
fi
```

The way of how to get the mic status is highly dependent on the system. For some the variant before your change might be the only one working, for others that might be your changed one.